src/components/ItaliaTheme/Blocks/HighlightedContent/Sidebar.jsx

```diff
@@ -34,6 +34,10 @@ const messages = defineMessages({
     id: 'show_type',
     defaultMessage: 'Mostra il tipo',
   },
+  show_data: {
+    id: 'show_data',
+    defaultMessage: 'Mostra la data',
+  },
```

Comment:
Aggiungendo una nuova stringa, va aggiunta anche ai file di traduzione in `locales`. Esegui `yarn i18n` e dovrebbe comparirti tutto

Reply:
Ho provato, il commando non funziona, ti avevo anticipato questo via slack.